client/package.json

```diff
@@ -1,18 +1,96 @@
 {
-  "name": "{{pkgName}}-root",
+  "name": "{{pkgName}}",
   "description": "",
   "version": "{{applicationVersionWeb}}",
   "private": true,
   "license": "UNLICENSED",
+  "config": {
+    "API_HOST": "{{apiHost}}",
+    "PORT": 6060,
+    "ENABLE_SSO": {{enableSSO}}
```

Comment:
nitpick:

can we have `"{{enableSSO}}"` here? Line looks like error + follow what we have above with `apiHost` (or others like `rootElement`)

Reply:
API host and root element are strings, while this is a boolean